WordPress/src/main/res/layout/stats_quick_scan_item.xml

```diff
@@ -32,6 +32,15 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:text="@string/unknown" />
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/start_text"
+            style="@style/StatsBlockText"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_vertical"
```

Comment:
np: Parent layout's height is height is `wrap_content`. So, I guess `android:layout_gravity="center_vertical"` has no fuction here. We can remove it. Also same for `end_text`.